csharp/samples/SampleReceiver/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace SampleReceiver
+{
+    using System;
+    using System.Threading.Tasks;
```

Comment:
Can we call this sample as EventProcessorHostReceiver. I really want to have a low level API receive sample as well.